app/controllers/stories_controller.rb

```diff
@@ -1,4 +1,6 @@
 class StoriesController < ApplicationController
+  include CommentsHelper
```

Comment:
**Suggestion:** Instead of including the comments helper maybe this is better suited to be managed by a `StoriesHelper`?

Another idea could be that if this include is only required for a single call then it could be worth making this a class method instead? This way we can avoid the include and make the call like this:

```ruby
comment_count = CommentsHelper.comments_to_render(params[:view])
```

Totally open to hear other team members opinion on this though

Reply:
+1 to theses ideas!

Reply:
Yes, seems a bit strange to include a CommentHelper in Stories controller, but it is because we do not get  there via the comment controller. Though, I feel these methods should belong to the comment helper.

But too get rid of this include,  yes, I can change the call with a static method.

Reply:
I think the best solution here is to use the builtin `helpers` proxy, the call would become `helpers.comments_to_render(params[:view])`

Reply:
Did not know that. Settled :)